Gemfile

```diff
@@ -28,3 +28,7 @@ end
 group :xdb do
   gem 'mysql2', '~> 0.3.16'
 end
+
+group :zk do
+  gem 'zk'
```

Comment:
```Gemfile.lock``` pegs the version at 1.9.5.

Updating the version of the zk gem requires a PR to update the version in ```Gemfile.lock``` and commit that change.

```
bundle update zk
git commit -a -m 'Updating zk gem'
```